modules/cloud/pom.xml

```diff
@@ -33,7 +33,7 @@
     <url>http://ignite.apache.org</url>
 
     <properties>
-        <jcloud.version>1.9.0</jcloud.version>
+        <jcloud.version>2.0.0</jcloud.version>
```

Comment:
Earlier <jcloud.version> was same for every dependencies. But with latest version upgrade allcompute dependency is 2.0.0 and others are 1.9.3.
That was the reason of change.